src/transactions/feed/TransactionFeed.test.tsx

```diff
@@ -18,6 +18,11 @@ import { mockCusdAddress } from 'test/values'
 
 jest.mock('src/statsig', () => ({
   getFeatureGate: jest.fn(() => false),
+  getDynamicConfigParams: jest.fn(() => ({
```

Comment:
Noticed these in a bunch of test files, are the mocks on these test files required because there's a nested dependency to this dynamic config? And is the getFeatureGate mock now required? I wonder if we should consider creating a shared mock under `__mocks__` similar to how navigate / analytics is mocked so we don't need this everywhere

Reply:
That's correct, re: nested dependency. And yes, the `getFeatureGate` mock is still required here. And interesting -- I haven't messed around with shared mocks before. How would that work? Would there be a mock for the entirety of `src/statsig` that would automatically be included in all tests? What if we want to override this mock (which will certainly be the case)?

Reply:
yes it would mock `src/statsig` for all tests. We can still setup test specific mocks as we do normally and it will override this mock. In any case its outside the scope of this PR, but something to consider if we have more of these shared mocks.